Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN addgroup -S frinx && adduser -S frinx -G frinx
 USER frinx
 
 COPY --from=builder /go/bin/schellar .
-ADD schellar/migrations/ ./migrations
+COPY schellar/migrations/ ./migrations
+
+HEALTHCHECK --interval=10s --timeout=5s --start-period=20s --retries=5 CMD [ "wget --spider -q 127.0.0.1:3000/liveness" ]
```

Comment:
Using healtcheck in dockerfile is maybe not good option. In one book that I read, there was a statement like this: "While using HEALTHCHECK is a great option, you should avoid using it to run your containers on Kubernetes or a similar container orchestrator. You should make use of liveness and readiness probes instead. Similarly, you can define health checks on Docker Compose if you are using it, so use that instead of baking the health check into the container image." Modern DevOps Practices, Gaurav Agarwal

Reply:
> You should make use of liveness and readiness probes instead

That's what we are doing aren't we ? Calling /liveness of the app

Reply:
Yes, we are doing that properly right now ... but binding that heatcheck into docker image like this is not a good idea if I understand that statement properly
![image](https://user-images.githubusercontent.com/32100265/168273766-7b003460-500b-44ce-9ffa-046e0b8def2b.png)

Reply:
I think is better to have some default healthcheck, which can be overridden in the orchestrator, than no healtcheck. 
But at this time, healtcheck was removed from PR.